tools/benchmark-query-engine/main.go

```diff
@@ -37,15 +37,15 @@ type app struct {
 	tempDir             string
 	dataDir             string
 	binaryPath          string
-	cpuProfilePath      string
-	memProfilePath      string
 	ingesterAddress     string
 	cleanup             func()
 
 	count           uint
 	testFilter      string
 	listTests       bool
 	justRunIngester bool
+  cpuProfilePath  string
```

Comment:
linter is going to be grumpy about this

Reply:
Serves me right for trying to resolve the merge in the GitHub UI rather than locally 😄